http/watcher.go

```diff
@@ -140,6 +130,17 @@ func (w *BTCAddressWatcher) performRegisterPegIn(txHash string) error {
 	if err != nil || !s {
 		return fmt.Errorf("transaction failed. hash: %v", tx.Hash())
 	}
-
+	h, err := w.rsk.HashQuote(w.quote)
+	if err != nil {
+		return err
+	}
+	hb, err := hex.DecodeString(h)
+	if err != nil {
+		return err
+	}
+	err = w.lp.RefundLiquidity(hb)
+	if err != nil {
+		return fmt.Errorf("failed to refund to liquidity provider. quote: %v. error: %v", h, err)
+	}
```

Comment:
I would put this outside this function. In the same place where close(w.done) is called.

Reply:
done.